src/docs/arch-connection.dox

```diff
@@ -0,0 +1,55 @@
+/*! @arch_page arch-connection Connection
+
+@section arch_conn_def Definition
+
+A connection is a handle to a WiredTiger database instance and only one connection can be opened at
+a time. Internally, a connection is represented by \c WT_CONNECTION.
+
+@section arch_conn_lifecycle Life cycle
+
+@subsection arch_conn_init Initialization
+
+A connection is initialized using WT_CONNECTION::wiredtiger_open. Given a set of configuration
+items (see @ref database_config), specific WiredTiger features can be enabled and their behavior
```

Comment:
I guess add something on lines of -> wiredtiger_open accepts a list of configuration items that can be used to enable different WiredTiger features ...

Reply:
See 81955014b367084664e0a3e42283ff09a2839bac